hazelcast/src/test/java/com/hazelcast/topic/TopicTest.java

```diff
@@ -627,4 +629,76 @@ public void onMessage(Message msg) {
         assertEquals(1000, stats.getPublishOperationCount());
         assertEquals(2000, stats.getReceiveOperationCount());
     }
+
+    @Test
+    @SuppressWarnings("unchecked")
+    public void testTopicMultiThreading() throws Exception {
```

Comment:
Just a note: If that method if important for code coverage, please use `SlowTest`. If it's just for correctness or a kind of integration/stress test use `NightlyTest`. All categories will be executed, but just `SlowTest` is considered for our code coverage reports.